tests/unit/specs/components/common/__snapshots__/MaintenanceBar.spec.js.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`MaintenanceBar has the expected html structure 1`] = `<!---->`;
+exports[`MaintenanceBar has the expected html structure 1`] = `<div />`;
```

Comment:
The snapshot would be more useful if some maintenance messages are loaded first.

Reply:
Updated snapshot:

```
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaintenanceBar has the expected html structure 1`] = `
<div>
  <div>
    <bar-stub
      show="true"
      type="success"
    >
      success test message
    </bar-stub>
  </div>
</div>
`;
```